components/PostCard.tsx

```diff
@@ -16,7 +17,16 @@ const PostCard = ({ post }: Props) => {
         <div className="content">
           <Link href={`/post/${post.slug}`} as={undefined}>
             <a className="post-img">
-              <img src={post.featuredImage} alt="blog post image" />
+              {/* <img src={post.featuredImage} alt="blog post image" /> */}
+              <Image
+                src={post.featuredImage}
+                alt="blog post image"
+                quality={100}
+                layout="fill"
+                objectFit="cover"
+                width={800}
+                height={400}
```

Comment:
@ntorbinskiy why those values in here?

Reply:
@1alexvash 
They're nonsense here, deleted